### DISCUSSION TITLE
Restore backup

### DISCUSSION THREAD

**Discussion by ChristopherMarie** [Ride-The-Lightning/RTL#959] [Troubleshooting Questions] [answered]
I am using mynode and would like to do seed restoration + upload the channel backup file with RTL.
I tried the following code to copy/paste the backup file into the folder however it returned Permission Denied:
scp <folderpath>\channel-all.bak admin@mynode.local:/mnt/hdd/mynode/rtl_backup/restore

Is there something incorrect with the command line or is there something I need to do with myNode to transfer the file?

Also, if I have already tried doing a seed backup using myNode's channel backup file, though it does not seem to be working properly. If I use RTL's backup file as well, might it complicate things?

**Comment by saubyk** [comment, marked as answer]
Hello @ChristopherMarie 
the copy command that you used doesn't seem right. Make sure of two things:
1. `restore` folder should be present
2. Your ID has permission to write (you can try to run the command with super user privilege or sudo)

To restore the funds from your node you need two things:
1. ensure that the lightning node has been recovered with the backup seed
2. the channel backup file used to restore funds is the one which has been generated using RTL
